hq/app/views/fragments/statusSummary.scala.html

```diff
@@ -0,0 +1,14 @@
+@import model.CredentialReportDisplay
+@import logic.ReportDisplay
+@import model._
+@(report: CredentialReportDisplay)
+
+@if(ReportDisplay.reportStatusSummary(report).contains(Red)) {
+    <i class="material-icons right red-text" data-position="bottom" data-delay="50">error</i>
+}
+@if(ReportDisplay.reportStatusSummary(report).contains(Amber)) {
```

Comment:
Make sure the design supports showing multiple reports if Red and Amber both exist.

Reply:
it does.